src/components/Form/WysiwygField.js

```diff
@@ -10,14 +10,15 @@ import WysiwygControls from 'components/Form/WysiwygControls'
 import ValidationError from 'components/Form/ValidationError'
 import HelpText from 'components/Form/HelpText'
 import FieldLabel from 'components/Form/FieldLabel'
+import { formatHTMLForTiptap } from 'common/utils/stringHelpers'
 
 function WysiwygField({
   label = '',
   nameId,
   errors,
   helpText,
   control,
-  toolbar = ['INLINESTYLES', 'BLOCKSTYLES', 'OL', 'UL', 'HEADER'],
```

Comment:
If we're going to remove this default list which included all the toolbar options, please add them to the propType checker:
`  toolbar: arrayOf(
    oneOf(['INLINESTYLES', 'BLOCKSTYLES', 'OL', 'UL', 'HEADER']),
  ),`

Reply:
Will do, good catch!